src/styles/variables.scss

```
@@ -0,0 +1,16 @@
+$cell-width: 100px;
+$gap: 1px;
+$set-padding: 10px;
+$border-color: #eee;
+$color-black: #000;
+$color-hover: plum;
+
+$days: (
+  "mon": "1",
```

```suggestion
  "mon": 1,
```
I guess it should a number, not string

---

It the reason why I had been had to use #{$i} syntax to calculate width. Thanx!